event/test/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+#----------------------------------------------------------------------------------------------
+# Add the executables
+#
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin/tests)
+
+add_executable(testCocktailGenerator testCocktailGenerator.cpp)
```

Comment:
Use cxx rather than cpp

Reply:
Fixed. Removed "cpp" file and added "cxx" file.